Permanent/Network/LegacyPlanningEndpoint.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  LegacyPlanningEndpoint.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 15.05.2023
+//
+
+import Foundation
+
+struct legacyPlanningArchiveDetails {
```

Comment:
move this to another file. It should be in the models folder. Also it should start with an Upper case letter.